pom.xml

```diff
@@ -76,8 +76,9 @@
         <graalvm.version>22.3.0</graalvm.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <quarkus.version>2.16.4.Final</quarkus.version>
-        <quinoa.version>1.2.7</quinoa.version>
+ 
+        <quarkus.version>2.16.5.Final</quarkus.version>
+        <quinoa.version>1.2.5</quinoa.version>
```

Comment:
Why downgrade Quiona?

Reply:
It is a wrong rebase; I'm sure that @shivam-sharma7 will fix it

Reply:
I think it's for `quarks 2.x` check release https://github.com/quarkiverse/quarkus-quinoa/releases

Reply:
ok, no problem